containers/layouts/ViewLayoutRadios.js

```diff
@@ -8,15 +8,17 @@ import inboxRowSvg from 'design-system/assets/img/pm-images/inbox-row.svg';
 
 const { COLUMN, ROW } = VIEW_LAYOUT;
 
-const ViewLayoutRadios = ({ viewLayout, onChange, loading }) => {
+const ViewLayoutRadios = ({ viewLayout, onChange, loading, id, ...rest }) => {
     const radioCardColumn = {
         value: COLUMN,
         checked: viewLayout === COLUMN,
         id: 'columnRadio',
         disabled: loading,
         name: 'viewLayout',
         label: c('Label to change view layout').t`Column`,
-        onChange: () => onChange(COLUMN),
+        async onChange() {
```

Comment:
No need `async` there.
```suggestion
        onChange() {
```

Reply:
oh, that's a typo...